pkgs/by-name/qq/qqmusic/package.nix

```diff
@@ -0,0 +1,140 @@
+{
+  fetchurl,
+  stdenv,
+  autoPatchelfHook,
+  makeWrapper,
+  lib,
+  makeDesktopItem,
+  copyDesktopItems,
+  # QQ Music dependencies
+  alsa-lib,
+  at-spi2-atk,
+  at-spi2-core,
+  cairo,
+  cups,
+  dbus,
+  expat,
+  gdk-pixbuf,
+  glib,
+  gtk3,
+  libdbusmenu,
+  libglvnd,
+  libpulseaudio,
+  mesa,
+  nspr,
+  nss,
+  pango,
+  pciutils,
+  udev,
+  xorg,
+  ...
+}:
+################################################################################
+# Mostly based on qqmusic-bin package from AUR:
+# https://aur.archlinux.org/packages/qqmusic-bin
+################################################################################
+let
+  libraries = [
```

Comment:
Maybe just move this to `buildInputs` and refer to it as `finalAttrs.buildInputs` during the install phase?